launcher/settingsView/csettingsview_moc.cpp

```diff
@@ -40,6 +40,26 @@ static const std::string cursorTypesList[] =
 	"software"
 };
 
+int findComboBoxItemByInternalName(QComboBox* comboBox, const std::string& targetInternalName)
```

Comment:
It doesn't find item, but index

Reply:
renamed to findComboBoxItemsIndexByInternalName